### PR TITLE
Fix and rebalance Fort Lonestar - prep-1512 version

### LINKED ISSUE #10100: No build progress on Fort Lonestar
On playtest 20151115.

In a game of 3, I was bottom left (top left was missing) there was no build status bar on my barracks. I could see everyone else's.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/28584145-no-build-progress-on-fort-lonestar?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by abcdefg30**
#10205 for prep.

Fixes #10100.

### CODE REVIEW

**Comment by obrakmann** [comment]
looks ok, mostly. I think I'm going to be a party pooper and nak the giant dog and the mini 3tnks.

File: mods/ra/maps/fort-lonestar/map.yaml [hunk 1]
```diff
@@ -563,16 +577,31 @@ Rules:
...
 			Period: 250
 			Amount: 50
+	MOBILETENT:
+		Inherits: MCV
+		Transforms:
+			IntoActor: tent
+			Offset: 0,0
+		RenderSprites:
+			Image: MCV
+			Scale: 0.5
```
Comment (GraionDilach) [inline mods/ra/maps/fort-lonestar/map.yaml]: `RenderSprites`->`Scale` is a bad idea and should never be used. It just promotes #3031 (and prolly other issues as well).

Comment (MustaphaTR) [inline mods/ra/maps/fort-lonestar/map.yaml]: It's a MCV. do it even have a death anim?

Comment (GraionDilach) [inline mods/ra/maps/fort-lonestar/map.yaml]: That's irrelevant. This is just bad coding based on a legacy option.

The new `WithDecoration` proposal denies the use of that trait for WW-styled `EMP`/mindcontrol/`AttachEffect.Animation` usecases - forcing `WithIdleOverlay` for such which gets affected by `Scale` and ultimately leads into severe issues due to some passive encouragement from usecases like here.

**Comment by abcdefg30** [comment]
Seems like I forgot to mention that I pushed an update...

So, anyway: Updated.

**Comment by obrakmann** [comment]
Thanks. Looks fine to me now. :+1:

**Comment by abcdefg30** [comment]
Sorry to have to bother you again, but I had to add 4 new commits to cover the newest set of issues.

**Comment by Mailaender** [comment]
Awesome. :white_check_mark:

**Comment by Mailaender** [comment]
Already in https://github.com/OpenRA/OpenRA/wiki/Changelog/

**Pull request merged**